con-master/src/main/resources/application.properties

```diff
@@ -3,7 +3,10 @@ spring.datasource.username=root
 spring.datasource.password=root
 spring.datasource.driver-class-name=com.mysql.jdbc.Driver
 
+
+cors.allowed.origins=http://localhost:3000
+
+image.directory.path=C:/var/concierge/images/
```

Comment:
You probably shouldn't use `var`. `var` is a well known directory which exists in root directory of Unix based systems.